src/NeXus.def

```diff
@@ -0,0 +1,84 @@
+LIBRARY NeXus.dll
```

Comment:
It's an alternative to dllimport/dllexport - it's what napi has used in the past, so I just readded it to avoid having to add dllimport/dllexport to napi.h / napi.c